osi_object.proto

```diff
@@ -1062,18 +1073,191 @@ message MovingObject
+        // should be followed for identifying bones.
+        //
+        message Bone
+        {
+            // Bones are identified by their type, combined with which body side
+            // they are on.
+            //
+            // To properly identify the bones the pre-defined naming convention
+            // must be used.
+            //
+            // \image html OSI_SkeletonNamingConvention.svg
+            //
+            // A bone's name, position, and orientation is defined by the end-point closer
+            // to the root. For example: the "LOWER_ARM_L" will define the point in the left
+            // elbow of the model.
+            //
+            // If a bone which is more than one layer detached from the
+            // root point is used, all bones between that bone and the root also
+            // need to be defined in order to create a complete chain!
+            //
+            // If information about bones are missing, they may be left empty.
+            //
+            optional Type type = 1;
+
+            // Position of the bone.
+            //
+            // Reference System is the root, defined by bbcenter_to_root
+            // (\c PedestrianAttributes::bbcenter_to_root).
+            //
+            optional Vector3d position = 2;
+
+            // Orientation of the bone.
+            //
+            // Reference System is the root, defined by bbcenter_to_root
+            // (\c PedestrianAttributes::bbcenter_to_root).
+            //
+            optional Orientation3d orientation = 3;
+
+            // Orientation of the bone.
+            //
+            // Reference System is the root, defined by bbcenter_to_root
+            // (\c PedestrianAttributes::bbcenter_to_root).
+            //
+            optional float length = 4;
+
+            // Determines a bone is explicitly missing from the pedestrian model.
+            //
+            // In case a pedestrian has missing limbs this can be explicitly communicated
+            // by setting this boolean to TRUE.
+            //
+            optional bool missing = 5;
+
+            // The type of the bone.
+            //
+            // \note Bone of each type can be assigned, or left empty depending
+            // on the desired level of detail, or present data. However, if a
+            // bone is defined, all bones in the chain from that bone back to
+            // the root point must be defined to create a complete chain.
+            //
+            // \image html OSI_PedestrianModelHierarchy.jpg
+            //
+            enum Type
+            {
+                // Root point of the pedestrian. Defined by bbcenter_to_root
+                // (\c PedestrianAttributes::bbcenter_to_root).
+                //
+                TYPE_ROOT = 0;
+
+                // Bone defining the hip.
+                //
+                TYPE_HIP = 1;
+
+                // Bone defines the lower part of the spine.
```

Comment:
I suggest to always use defining, rather than defines, in this list...
```suggestion
                // Bone defining the lower part of the spine.
```